ArduCopter/Copter.cpp

```diff
@@ -485,11 +485,18 @@ void Copter::update_batt_compass(void)
 }
 
 // Full rate logging of attitude, rate and pid loops
-// should be run at 400hz
-void Copter::fourhundred_hz_logging()
+// should be run at loop rate
+void Copter::loop_rate_logging()
 {
     if (should_log(MASK_LOG_ATTITUDE_FAST) && !copter.flightmode->logs_attitude()) {
         Log_Write_Attitude();
+        Log_Write_PIDS(); // only logs if PIDS bitmask is set
+    }
+    if (should_log(MASK_LOG_FTN_FAST)) {
+        AP::ins().write_notch_log_messages();
+    }
+    if (should_log(MASK_LOG_IMU_FAST) && !(should_log(MASK_LOG_IMU))) {
```

Comment:
Don't think this is right. Fast logging should take preference

Reply:
your right....will fix